domains/realyash.json

```diff
@@ -0,0 +1,13 @@
+  {
+    "owner": {
+        "username": "Real-Yash",
+        "email": "tempyuva00@gmail.com"
+        "repo": "https://github.com/Real-Yash/yash",      
+    },
+    "record": {
+        "CNAME": "real-yash.github.io"
+    }
+  "link":{
+        "link":"https://real-yash.github.io/yash/"
+  } 
```

Comment:
?????
This is very VERY invalid, please remove it

Reply:
is this okeyy??